### DISCUSSION TITLE
How can I use client certificate validation with Gateway API?

### DISCUSSION THREAD

**Discussion by h-otter** [projectcontour/contour#5379] [Q&A] [answered]
[HTTPProxy of `projectcontour.io/v1` supports client certificate validation.](https://projectcontour.io/docs/1.25/config/tls-termination/#client-certificate-validation)
Is there a way to validate client certificates using the Gateway API?
If possible, I would like to configure GatewayClass to enforce client certificate validation, like `globalExtAuth` in ContourConfiguration.

Thank you!!

**Comment by skriss** [comment, marked as answer]
@h-otter this is not currently possible with Gateway API; here's the upstream tracking issue I found: https://github.com/kubernetes-sigs/gateway-api/issues/91
